rust/core/Cargo.toml

```diff
@@ -13,3 +13,46 @@ derivative = "2.2.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.57"
 schemars = "0.8.8"
+
+cryptoxide = "0.4.2"
+ed25519-bip32 = "0.4.1"
+sha2 = "^0.9"
+digest = "^0.9"
+bech32 = "0.7.2"
+hex = "0.4.0"
+itertools = "0.10.1"
+getrandom = { version = "0.2.3", features = ["js"] }
+rand = "0.8.5"
+fraction = "0.10.0"
+base64 = "0.13"
+num-bigint = "0.4.0"
+num-integer = "0.1.45"
+#rand_os = "0.1"
+thiserror = "1.0.37"
+cfg-if = "1"
+
+# These can be removed if we make wasm bindings for ALL functionality here.
+# This was not done right now as there is a lot of existing legacy code e.g.
+# for Byron that might need to be used from WASM and might not.
+# We can remove this dependency when that is decided.
+#
+# The other use-case here is enums. Without this two enums would need to be defined
+# despite wasm_bindgen supporting C-style enums (with non-negative values) 100%
+# This could possibly be resolved with macros but maybe not.
+
+# non-wasm
```

Comment:
It seems we will have to use the full wasm_bindgen not the noop_proc_macro here as in WASM builds the `core` crate will get compiled using `noop_proc_macro` even when it's only used as a dependency by the `wasm` crate in a WASM build.